warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -153,6 +151,9 @@ public class QueryOptions implements OptionDescriber {
     public static final String GROUP_FIELDS = "group.fields";
     public static final String GROUP_FIELDS_BATCH_SIZE = "group.fields.batch.size";
     public static final String UNIQUE_FIELDS = "unique.fields";
+    public static final String MOST_RECENT_UNIQUE = "most.recent.unique";
```

Comment:
The fields MOST_RECENT_UNIQUE and UNIQUE_CACHE_BUFFER_SIZE were removed in commit  2c2de3c0d0. Please delete them here.